packages/react-date-time/src/components/Calendar/CalendarYear/CalendarYear.base.tsx

```diff
@@ -267,8 +267,8 @@ const CalendarYearNavArrow = React.forwardRef(
         <Icon
           iconName={
             (direction === CalendarYearNavDirection.Previous) !== getRTL()
```

Comment:
Was this a regression compared to 7? If so, I suspect that this check being a bit convoluted (if technically the shortest way to get the desired result) contributed. So it might be helpful to either add a comment explaining the check, or expand it into something more verbose but clearer.

Reply:
pulled the check into two parts and moved it above the render, should be more obvious now. It looks like it was a regression